nixos/modules/services/printing/cupsd.nix

```diff
@@ -172,7 +172,7 @@ in
 
       webInterface = mkOption {
         type = types.bool;
-        default = true;
+        default = false;
```

Comment:
I've checked and both Arch Linux and Fedora keep this as true.
Didn't check formally in ubuntu but IIRC they keep it on.

Any extra reasoning why we should change this? It would be unexpected compared to everywhere else.

Reply:
See: [Always Secure Your localhost Servers](https://davywybiral.blogspot.com/2019/05/always-secure-your-localhost-servers.html)

At a minimum, whether or not you are running the cupsd web interface is available knowledge to any website you visit (or any ad shown on that website).  Since cupsd runs as root and isn't sandboxed and doesn't drop privileges, that web interface would be very interesting to any local attacker who wants to gain root.

Personally I leave it off unless I actually need to use it.

Reply:
Thought there was some sort of security implication. I'd say it would be a good idea to mention that in the release note.

Reply:
Yeah, perhaps we can leave it as is. I didn't realize that other distros did the same thing. My opinion is that it's usually best to enable as little as possible and the web interface is definitely not needed for normal cups usage

Reply:
> Since cupsd runs as root and isn't sandboxed and doesn't drop privileges, that web interface would be very interesting to any local attacker who wants to gain root.

I believe in nixos cupsd has its own user group?

Interesting that it's disabled default on darwin
* https://github.com/apple/cups/blob/58b64dbba8d459b79bbc30302ce461276b0628dc/config-scripts/cups-defaults.m4#L430

At any rate, I don't think we should block this PR on whether or not the `WebInterface` should be locked down in some manner if left enabled. I'm guessing that's enabled on other distros because they didn't bother to change the default for any particular reason.